### ISSUE TITLE
More sample graphs

### ISSUE THREAD

**Issue by dkuppitz** [PommeVerte/gremlin-bin#20]
Would be nice to have a little more complex graphs. Could you add the Grateful Dead and Northwind graphs?

**Comment by dmill-bz** [comment]
Yeah this is on my todo list (short term) it's dependent on #15 though
